modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/css/main.scss

```
@@ -4,6 +4,10 @@
 	display: none;
 }
 
+.date-picker-hidden {
+	display: none !important;
+}
+
```

Hi @pedro-oliveira446, I tested locally and it works without the `!important`. Please, could you remove it? We need to avoid using this property. Thanks!

```suggestion
.date-picker-hidden {
	display: none;
}

```

---

Done @alinedoleron !